src/plugins/custom_import_map/common/constants/shared.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const BASE_NODE_API_PATH = '/_plugins/geospatial';
+export const UPLOAD_GEOJSON_API = `${BASE_NODE_API_PATH}/geojson/_upload`;
+export const FILE_PAYLOAD_SIZE = 26214400; // 25 MB
```

Comment:
is there a way to extract this from 25 MB? If we decided to increase this limit, it will be better if we have to update in one place than two places. WDYT?